.circleci/config.yml

```diff
@@ -169,7 +169,37 @@ commands:
               echo "This is not a pull request, skipping..."
             fi
 
-
+  upload_binary_size_for_android_build:
+    description: "Upload binary size data for Android build"
+    parameters:
+      build_type:
+        type: string
+        default: ""
+      docker_image:
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: Upload << parameters.build_type >>
+          no_output_timeout: "5m"
+          command: |
+            docker_image="<< parameters.docker_image >>"
+            echo "docker-image: $docker_image"
+            export id=$(docker run --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -t -d -w /var/lib/jenkins ${docker_image})
```

Comment:
@ezyang inspired by your comment at: https://github.com/pytorch/pytorch/pull/40199/files#r443661066 - do I need run `docker pull ${docker_image}` here explicitly? I was thinking the command runs immediately after the previous command so we can assume it runs on the same machine and can reuse the docker image locally.

Do you think we should merge this "upload_binary_size_for_android_build" command into android build command instead?

Reply:
If they're in the same steps, I think all of the caches stick around.